MapboxGeocoder/MBPlacemark.swift

```diff
@@ -125,8 +125,10 @@ open class Placemark: NSObject, Codable {
         }
     }
     
-    @objc open override var hashValue: Int {
-        return identifier.hashValue
+    open override var hash: Int {
```

Comment:
These are generated by the compiler and not needed in Swift 4.2. Can we keep the old `hashValue` but wrap it in a 

```swift
#if swift(>=4.2)
#else
#endif
```